test/evergreen.yml

```diff
@@ -262,14 +262,17 @@ functions:
           DEFINED_EVERGREEN_CONFIG_FLAGS=${DEFINED_EVERGREEN_CONFIG_FLAGS/\-DHAVE_BUILTIN_EXTENSION_ZSTD=1/}
         fi
 
-        if [[ "${build_variant|}" =~ "macos-13" ]]; then
+        if [[ "${build_variant|}" =~ "macos-14" ]]; then
```

Comment:
It seems we only use macos-14-arm64, it might be worth considering to change the existing code to that. I wonder what happened.

Reply:
Update: https://jira.mongodb.org/browse/WT-12595 I have found the reason why. Disregard this comment.